packages/slice-machine/src/theme.ts

```diff
@@ -765,8 +768,8 @@ const AppTheme = (): Theme =>
       },
       inlineCode: {
         fontFamily: "monospace",
-        color: "error",
-        bg: "muted",
+        color: "#6E56CF",
+        bg: "#F1EEFE",
```

Comment:
Why this change? Would be good to avoid string literal colours outside the colours section at the top

Reply:
I agree but the whole theme needs to be refactored, and I can't say for sure yet what belongs in colors and what does not